index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="./assets/styles/style.css" />
+    <title>Codeeq Microverse Personal Portfolio Project</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="my-logo">My Logo</h1>
+      <div class="btn-scale">
+        <img src="./assets/images/Union.svg" alt="nav icon" />
+      </div>
+    </header>
+    <main>
+      <section class="hero">
+        <h2>Hey There. I’m Sodeeq.</h2>
```

Comment:
- [ ] Please follow the `HTML` semantics more correctly. As I can understand you used the h1 for a logo but now as I mentioned everything in my above comment but here please use `h1` over `h2` because it is the main and first heading of your portfolio. Kindly change it because it will also improve your *A11y* accessibility of the website.